src/game/option_passport.c

```diff
@@ -97,6 +98,16 @@ static void Option_PassportInitSaveRequester(int16_t page_num);
 static void Option_PassportLoadGame(void);
 static void Option_PassportSelectLevel(void);
 
+void Option_PassportInitSavegameStrings(void)
+{
+    m_SavegameStrings = Memory_Alloc(g_Config.maximum_save_slots * sizeof(char *));
+    for (int i = 0; i < g_Config.maximum_save_slots; i++) {
+        m_SavegameStrings[i] = Memory_Alloc(sizeof(char *));
```

Comment:
That doesn't make sense – we're now allocating 4 bytes for each string. Please just use `MAX_LEVEL_NAME_LENGTH` as it used to be the case.